ui-test-example/build.gradle

```diff
@@ -44,8 +44,14 @@ intellij {
 runIdeForUiTests {
     systemProperty "robot-server.port", "8082"
     systemProperty "ide.mac.message.dialogs.as.sheets", "false"
+
+    // enable here and in test block - to log the retrofit HTTP calls
+    // systemProperty "debug-retrofit", "enable"
```

Comment:
I think there is no need to add a property on server side. In `test` should be enough

Reply:
Done

Reply:
And tested - you were right there is no need.